packages/models/src/store/store.ts

```diff
@@ -240,6 +268,7 @@ export class Store<
   private serializeField(type: StorageLocation, offChainValue: unknown) {
     if (!this.schemaOption || typeof this.schemaOption !== 'object') return { hexString: '0x', offset: 0, length: 0 }
     this.assertStorage(this.getStorage(type))
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
> Better to avoid non-null assertion

It is not the change for this PR, I just add this to avoid lint warning. I think maybe it is better to changing in another PR.